snakemake_executor_plugin_kubernetes/__init__.py

```
@@ -351,15 +452,16 @@ async def check_active_jobs(
+                    self.logger.debug(f"Job {j.external_jobid} is still active.")
                     yield j
 
     def cancel_jobs(self, active_jobs: List[SubmittedJobInfo]):
         # Cancel all active jobs.
-        # This method is called when Snakemake is interrupted.
-
         for j in active_jobs:
             self._kubernetes_retry(
-                lambda: self.safe_delete_pod(j.external_jobid, ignore_not_found=True)
+                lambda: self.safe_delete_pod(
+                    j.external_jobid, ignore_not_found=True
```

Ruff (0.8.2)

Function definition does not bind loop variable `j`

(B023)